helloworld-consumer/src/main/java/KafkaConsumerExample.java

```diff
@@ -0,0 +1,59 @@
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.LogManager;
+
+import java.util.Collections;
+import java.util.Properties;
+
+public class KafkaConsumerExample {
+    private static final Logger log = LogManager.getLogger(KafkaConsumerExample.class);
+
+    public static void main(String[] args) {
+        KafkaConsumerConfig config = KafkaConsumerConfig.fromEnv();
```

Comment:
I would factor out the building of `props` from a given `config` into a separate method (it could easily be a method of `KafkaConsumerConfig`, actually).